### DISCUSSION TITLE
Access EPSPs and IPSPs

### DISCUSSION THREAD

**Discussion by carolinafernandezp** [jonescompneurolab/hnn-core#878] [General]
Hi,

Is it possible to access the excitatory and inhibitory post synaptic potentials (EPSPs and IPSPs) in the model? If there isn't a set way to do that, is there a workaround that people have used?

**Comment by jasmainak** [comment]
@ntolley can help you here !

**Reply by ntolley** [reply]
hi @carolinafernandezp 

You can do this by running `dpl = simulate_dipole(net, record_isec='all')`

This will let you access currents for specific receptors (ampa, gabaa, gabab, nmda) on each compartment from the attribute `net.cell_response.isec[0]`

Let me know if that works for you!
